src/components/src/main/java/org/apache/jmeter/visualizers/backend/BackendListener.java

```diff
@@ -106,12 +105,9 @@
     // Name of the test element. Set up by testStarted().
     private transient String myName;
 
-    // Holds listenerClientData for this test element
     private transient ListenerClientData listenerClientData;
 
-    /**
-     * Create a BackendListener.
-     */
+    /** Create a BackendListener. */
```

Comment:
I'm -0.5 regarding the changes like that.

Multi-line javadoc allows to add new items/lines without touching of the previous contents.

The change of multiline javadoc to a single-line does not really add much, and it definitely hurts in terms of merge conflicts in the future.

Reply:
My rationale for this one (and most I change like this) was to remove the 2 lines of vertical space taken up by a fairly pointless comment. I probably should have just removed this particular comment.

You're right it's good to think about with future changes like this if they should remain multi-line if they are likely to be changed or added to, I don't think this one will cause much of a merge problem (but I accept it wouldn't be as easy).

Reply:
> was to remove the 2 lines of vertical space taken up by a fairly pointless comment

Then should we kill the comment altogether?

I agree there's not much sense in writing "dumb" javadocs like `BackendListener`

> I don't think this one will cause much of a merge problem (but I accept it wouldn't be as easy).

Consider someone would add a constructor parameter, they might want to improve the documentation, and it would blow.

If you just removed the comment, it would probably be better :)